sdk/eventhubs/ci.yml

```diff
@@ -32,6 +32,7 @@ extends:
       LiveTestTimeoutInMinutes: 120
       LineCoverageTarget: 27
       BranchCoverageTarget: 13
+      UseFederatedAuth: true
```

Comment:
Is this necessary since the default is true?

Reply:
yes, see previous comments and PR message